mw-webapp/src/logic/wayPage/wayStatistics/WayStatistic.tsx

```diff
@@ -95,13 +95,13 @@ export const WayStatistic = observer((props: WayStatisticProps) => {
   const startDate = dayReportsReversed[0] ? dayReportsReversed[0].createdAt : props.wayCreatedAt;
   const lastDate = props.dayReports[0] ? props.dayReports[0].createdAt : props.wayCreatedAt;
 
-  const startDateLastWeek = startDate <= lastWeekDate
-    ? lastWeekDate
```

Comment:
If you don't use  const need to delete it / Or move logic to const and use const below instead of DateUtils.getLAstDate....